frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -199,7 +199,7 @@ export const apiCreateQuestion = question => {
     MetabaseAnalytics.trackStructEvent(
       "QueryBuilder",
       "Create Card",
-      createdQuestion.query().datasetQuery().type,
+      createdQuestion.type(),
```

Comment:
@bshepherdson 
Not sure how else to send the question type through without this method.

Reply:
Ah, I think that's an acceptable use of `.type()`, when we're logging it.